TACC/Core-Styles
# chore(components): align, remove default max width

## Overview

No default max. width for `align` component.

## Related

Skipped.

## Changes

- moved `max-width: 50%` from `align.css` to `align/demo.css`
- added comments about the change

## Testing

1. Open "Components" > "Align".
2. Confirm `max-width: 50%` is applied to `.align-center` via `demo.css`, and **not** any other stylesheet.

## UI

![localhost_3000_components_preview_align](https://user-images.githubusercontent.com/62723358/217347715-ddda1c29-7e57-4ea5-b0bf-56662bed403e.png)

## Notes

Downstream clients are not affected, because only TACC/Core-CMS knew about this, and it overrides it with its own value. I'll update Core-CMS to not consider that an overwrite.
